### PR TITLE
Angular resources

### PR DESCRIPTION
**Pull request by chhex**
@crhenrici  I changed the Build of SSServer to exclude the Client jar in the bootJar, but have it unpacked with the Distribution Plugin into a directory /web. This seems to be working. 

The start  scripts need to be adopted , i changed one with 2 possibilities..... and comment

Totally untested ;-)

### CODE REVIEW

File: SSServer/build.gradle [hunk 1]
```diff
@@ -13,20 +13,27 @@ java {
     targetCompatibility = JavaVersion.VERSION_1_8
```
Comment (chhex) [inline SSServer/build.gradle:13]: This are actually the changes that matter .... in the build.gradle

File: SSServer/src/main/java/com/prose/crhen/SSServer/MockDataInitializer.java [hunk 2]
```diff
@@ -8,7 +8,6 @@
 import org.slf4j.Logger;
```
Comment (chhex) [inline SSServer/src/main/java/com/prose/crhen/SSServer/MockDataInitializer.java:8]: Unimportant .... just adopted that it compiles

File: SSServer/src/main/resources/installation/start.sh [hunk 3]
```diff
@@ -3,4 +3,11 @@ export DB_USER=$1
 export DB_PW=$2
 export DB_URI=$3
 export DB_NAME=$4
-java -jar lib/SSServer-0.0.1-SNAPSHOT.jar --spring.config.location=config/ --spring.profiles.active=$5
\ No newline at end of file
+java -jar lib/SSServer-0.0.1-SNAPSHOT.jar --spring.config.location=config/ --spring.resources.static-locations=web/static --spring.profiles.active=$5
```
Comment (chhex) [inline SSServer/src/main/resources/installation/start.sh:6]: Here two possibilities one with the --spring.resources.static-locations=web/static path

File: SSServer/src/main/resources/installation/start.sh [hunk 4]
```diff
@@ -3,4 +3,11 @@ export DB_USER=$1
...
+
+oder
+
+java -Dloader.path="web/" -jar lib/SSServer-0.0.1-SNAPSHOT.jar --spring.config.location=config/  --spring.profiles.active=$5
```
Comment (chhex) [inline SSServer/src/main/resources/installation/start.sh:10]: The other with the -Dloader.path="web/" path

File: SSServer/src/main/resources/installation/start.sh [hunk 5]
```diff
@@ -3,4 +3,11 @@ export DB_USER=$1
...
+
+
+ACHTUNG : untested stuff
```
Comment (chhex) [inline SSServer/src/main/resources/installation/start.sh:13]: I haven't tested these variant. Maybe if you have time. Good luck ;-)

**Comment by crhenrici** [comment]
cool thx, will try it out...

**Pull request merged**